File: Project Milestone 2/Mihir/Python/topic.py
from kafka.admin import KafkaAdminClient, NewTopic


admin_client = KafkaAdminClient(bootstrap_servers=["localhost:9093","localhost:9094","localhost:9095"])

topic_list = NewTopic(name="cloud", num_partitions=3, replication_factor=3)

admin_client.create_topics([topic_list])

print ("Successfull created topic name: cloud")

File: Project Milestone 2/Mihir/Kafka/topic.js
//const Kafka = require("kafkajs").Kafka
const {Kafka} = require("kafkajs")

run();
async function run(){
    try
    {
         const kafka = new Kafka({
              "clientId": "myapp",
              "brokers" :["localhost:9093","localhost:9094","localhost:9095"]
         })

        const admin = kafka.admin();
        console.log("Connecting.....")
        await admin.connect()
        console.log("Connected!")
        //A-M, N-Z
        await admin.createTopics({
            "topics": [{
                "topic" : "topic",
                "numPartitions": 2
            }]
        })
        console.log("Created Successfully!")
        await admin.disconnect();
    }
    catch(ex)
    {
        console.error(`Something bad happened ${ex}`)
    }
    finally{
        process.exit(0);
    }


}